accounts/views.py

```diff
@@ -61,6 +61,9 @@ def signup(request):
         business = Business(user=user)
         business.save()
 
+        # Create Plan
```

Comment:
The plan should be created by the Site Administrator prior to system usage, They should create the plan and set the price they want.